### PR TITLE
Factor out code references to GD to a standalone file

### PR DESCRIPTION
**Pull request by juliexxia**
Also minimal docs here

### CODE REVIEW

File: docs/authenticate.js [hunk 1]
```diff
@@ -179,12 +178,12 @@ class Authenticator {
...
             const message = 'Refresh token error: ' + response.status;
             console.error(message, response);
+            const contact = CONTACT ? CONTACT : OWNER;
             // TODO(#395): Find GD contact to list here.
             alert(
                 'Error contacting server for access without EarthEngine ' +
                 'whitelisting. Please reload page and log in with an ' +
-                'EarthEngine-whitelisted account or contact ' +
-                'technology.manager@givedirectly.org ' +
+                'EarthEngine-whitelisted account or contact ' + contact + ' ' +
```
Comment (janakdr) [inline docs/authenticate.js]: need to ALL-CAPS contact?

File: token_server/token_server.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {createServer} from 'http';
 import parseBody from 'urlencoded-body-parser';
 import {storeGoogleCredentials} from './aws_get_credentials.js';
 import {generateEarthEngineToken} from './ee_token_creator.js';
+import {TOKEN_SERVER_ORIGINS} from '../docs/owner.js';
```
Comment (janakdr) [inline token_server/token_server.js]: I think deploying token server doesn't work with local files outside this directory, sadly :(

File: docs/authenticate.js (hunk 1 above)
Comment (juliexxia) [reply docs/authenticate.js]: I variablized it outside

File: token_server/token_server.js (hunk 2 above)
Comment (juliexxia) [reply token_server/token_server.js]: Sad, moved to a file in this dir

**Pull request merged**